fabric_plugin/tasks.py

```diff
@@ -351,9 +374,17 @@ def run_script(ctx,
 
     with ssh_connection(ctx, fabric_env) as conn, \
             _RemoteFiles(conn, process.get('base_dir')) as files:
+        run = conn.sudo if use_sudo else conn.run
         files.upload_script(local_script_path)
         fabric_ctx = _FabricCtx(ctx, files)
 
+        # handle execution python scripts
+        python_prefix = None
+        if files.is_py_script(local_script_path):
+            py_version, py_path = _check_python_version(run)
+            if py_version == '3':
```

Comment:
not sure why 2 and 3 are different here, instead of just using the path

Reply:
Removed the whole method